contracts/modules/STO/Capped/CappedSTO.sol

```diff
@@ -76,6 +81,13 @@ contract CappedSTO is CappedSTOStorage, STO, ReentrancyGuard {
         return this.configure.selector;
     }
 
+    /**
+     * @notice This function will allow STO to pre-mint all tokens those will be distributed in sale
+     */
+    function allowPreMinting() external withPerm(ADMIN) {
+        _allowPreMinting(cap);
```

Comment:
Should this flag be added to `getSTODetails`?

Reply:
Fixed